specification/msi/resource-manager/Microsoft.ManagedIdentity/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -0,0 +1,544 @@
+            "$ref": "./examples/IdentityGet.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The requested identity.",
+            "schema": {
+              "$ref": "#/definitions/Identity"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "UserAssignedIdentities_Delete",
+        "description": "Deletes the identity.",
+        "x-ms-examples": {
+          "IdentityDelete": {
+            "$ref": "./examples/IdentityDelete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Deleted Identity.",
+            "schema": {
+              "$ref": "#/definitions/Identity"
+            }
+          },
+          "204": {
+            "description": "The specified identity does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Identity": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The id of the created identity."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the created identity."
+        },
+        "location": {
+          "type": "string",
+          "description": "The Azure region where the identity lives."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties associated with the identity.",
+          "$ref": "#/definitions/IdentityProperties"
+        },
+        "type": {
```

Comment:
rename this to "identityType". You are missing the top level property "type" which corresponds to the resource type. So please add that one and rename this to identityType

Reply:
![image](https://user-images.githubusercontent.com/1617146/33154163-3737afb0-cf9b-11e7-9f4b-7a55d12d459e.png)

This is what our current response looks like: the type property in Swagger corresponds to the type we return in the response. Renaming it to "identityType" in Swagger would mean we have to change it in the response as well, right?

Reply:
I think this is OK it looks like Gaurav thought this property was in the resource properties